lib/src/network/tcp.rs

```diff
@@ -728,8 +728,17 @@ impl ServerConfiguration {
   }
 
   pub fn remove_backend(&mut self, app_id: &str, backend_address: &SocketAddr, event_loop: &mut Poll) -> Option<ListenToken>{
-      // ToDo
-      None
+    let backend = self.backends.remove_backend(app_id, backend_address);
+
+    let front_token = self.fronts.remove(app_id);
```

Comment:
why are you removing the frontend? There can be multiple backends for one frontend

Reply:
Oh yeah srry, maybe we should remove the frontend when the last backend is remove ?

Reply:
we should remove the frontend when we receive the `RemoveTcpFront` message

Reply:
Okai but we still have to remove the backend in the [ApplicationListener.back_addresses](https://github.com/sozu-proxy/sozu/blob/a9ccb886bad8e2af43cac6a7185ae5ffadba3b6b/lib/src/network/tcp.rs#L592) ?

Reply:
yes